.github/workflows/macos-latest.yml

```diff
@@ -0,0 +1,36 @@
+
+name: macos-latest
+
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
+  pull_request:
+
+jobs:
+  perl:
+
+    runs-on: macOS-latest
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Perl
+        run: |
+          brew install perl
+          curl https://cpanmin.us | perl - App::cpanminus -n
```

Comment:
We could install cpanminus (`cpanm`) from Homebrew, https://formulae.brew.sh/formula/cpanminus#default